packages/markdown-docx/src/ToOOXMLVisitor/helpers.js

```diff
@@ -211,17 +227,25 @@ function wrapAroundDefaultDocxTags(ooxml) {
           <w:color w:val="1F4D78" w:themeColor="accent1" w:themeShade="7F" />
         </w:rPr>
       </w:style>
+      ${LINK_STYLE_SPEC}
     </w:styles>
   </pkg:xmlData>
   </pkg:part>
     `;
 
+    let relationshipOOXML = '';
+    relationships.forEach(
+        ({ id, destination }) =>
+            (relationshipOOXML += `<Relationship Id="rId${id}" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/hyperlink" Target="${destination}" TargetMode="External"/>`)
```

Comment:
@algomaster99 
The above is a single tag, not multiple tags.So should I break it ?

Reply:
It has multiple attributes. You can break on each attribute like you do in react.

But it might be syntactically incorrect because word is weird. If you are in a rush, you can let this change be as is.